test/ipns-pubsub.js

```diff
@@ -97,7 +97,13 @@ const subscribeToReceiveByPubsub = async (nodeA, nodeB, idA, idB) => {
   const topic = `${namespace}${base64url.encode(keys.routingKey.toBuffer())}`
 
   // try to resolve a unpublished record (will subscribe it)
-  await expect(last(nodeB.api.name.resolve(idA, { stream: false }))).to.be.rejected()
```

Comment:
You should be able to assert on errors with something like

```javascript
await expect(last(nodeB.api.name.resolve(idA, { stream: false }))).to.eventually.be.rejected.with(/was not found in the network/)
```

Reply:
but that only happens in the first test not the second

Reply:
If the order of the API calls affects how the tests run this should be made explicit in the test - an expectation to fail based on the number of invocations, or a different helper function or something like that.